apps/src/dance/ai/DanceAiModal.tsx

```diff
@@ -913,7 +917,7 @@ const DanceAiModal: React.FunctionComponent<DanceAiModalProps> = ({
               currentMode={mode}
               showFor={[Mode.RESULTS]}
               id="start-over-button"
-              onClick={() => handleStartOverClick(false)}
+              onClick={handleStartOverClick}
```

Comment:
The mildly confusing bit is that we _cannot_ hand in `data-usingheader = {false}` because the attribute just gets omitted.

In this case, we could always re-tool to use string values of true/false, but that's kinda frowned up. And it has potential to cause confusion later if somebody sees `"false"` and changes it to `false`, assuming it shouldn't be quoted.

Anyway, so it's not included here. That's why we have the `|| false` up in the click handler to explicitly set the value in case one wasn't handed in.